tests/gnrc_sock_dns/tests/01-run.py

```diff
@@ -274,6 +275,7 @@ def testfunc(child):
     tap = get_bridge(os.environ["TAP"])
     lladdr = get_host_lladdr(tap)
 
+    time.sleep(3)
```

Comment:
The value of three seconds also makes sense, as typically three NS are sent in an interval of 1s to do DAD.